ffmd-inventory

```diff
@@ -0,0 +1,31 @@
+---
+all:
+  vars:
+    users:
+    - logname: "tux"
```

Comment:
vielleicht sollte diese table direkt initialisiert sein mit jaspers und meinen Daten...

Reply:
ja, sollte sie. Aber:
1. Ich wollte erstmal sehen, ob das Prinzip überhaupt akzeptiert wird
2. jeder von euch kann da per PR seine Daten reinreichen und damit selbst entscheiden, was in der Tabelle stehen soll